ckanext/data_qld_theme/templates/package/read.html

```diff
@@ -2,7 +2,7 @@
 
 {% block meta %}
 {{ super() }}
-{% if c.pkg_dict['organization'] and c.pkg_dict['organization']['title'] %}
-  <meta name="DCTERMS.creator" content="c=AU; o=The State of Queensland; ou={{ c.pkg_dict['organization']['title'] }}" scheme="AGLSTERMS.GOLD" />
+{% if pkg['organization'] and pkg['organization']['title'] %}
```

Comment:
Is this inside resource_read that sets

```{% set pkg = c.pkg_dict or pkg_dict %}```

Reply:
Actually it comes from the `ckan_extends` directive at the top, which makes the template extend the equivalent core template. Core `ckan/templates/package/read.html` starts with:

        {% set pkg = c.pkg_dict %}